app/SearchApi/SearchRequest.Adaptor/Consumer/NotificationAcknowledgedConsumer.cs

```diff
@@ -30,9 +31,16 @@ public async Task Consume(ConsumeContext<NotificationAcknowledged> context)
             using (LogContext.PushProperty("RequestRef", $"{context.Message?.RequestId}"))
             using (LogContext.PushProperty("AgencyCode", $"{context.Message?.ProviderProfile?.Name}"))
             {
-                _logger.LogInformation("get the NotificationAcknowledged message.");
-                var cts = new CancellationTokenSource();
-                await _searchRequestNotifier.NotifySearchRequestEventAsync(context.Message.RequestId, context.Message, cts.Token, context.GetRetryAttempt(), _retryConfig.Value.RetryTimes);
+                try
+                {
+                    _logger.LogInformation("get the NotificationAcknowledged message.");
+                    var cts = new CancellationTokenSource();
+                    await _searchRequestNotifier.NotifySearchRequestEventAsync(context.Message.RequestId, context.Message, cts.Token, context.GetRetryAttempt(), _retryConfig.Value.RetryTimes);
+                }catch(Exception e)
+                {
+                    _logger.LogError("NotificationAcknowledged {requestRef} is put into error queue.", context.Message?.RequestId);
```

Comment:
Log the exception as the first parameter

Reply:
ok